platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/gossip/sync/config/SyncConfig.java

```diff
@@ -32,7 +32,8 @@
  */
 @ConfigData("sync")
 public record SyncConfig(
-        @ConfigProperty(value = "syncAsProtocolEnabled", defaultValue = "false") boolean syncAsProtocolEnabled,
+        @ConfigProperty(value = "syncAsProtocolgit chasdfasdfasdfasdfEnabled", defaultValue = "false")
```

Comment:
Something tells me you didn't intend to commit this

Reply:
Fixed. I also removed the `value = ...` arguments, since they are not necessary if the name matches the value string.